spec/requests/admin/administrators/administrators_spec.rb

```diff
@@ -97,6 +97,16 @@
         }),
       )
     end
+
+    it "sends new admin an account created email" do
+      mail = instance_double(ActionMailer::MessageDelivery, deliver_now: true)
+      url = "http://www.example.com/users/sign_in"
+      allow(AdminMailer).to receive(:account_created_email).and_return(mail)
```

Comment:
I know I introduced the `mail = ...` stuff, but I since found a better solution - `and_call_original` :)

Reply:
I did replace that, not sure why it's still there

Reply:
Forgot to push the latest, thanks